advanced/am-pattern-2/templates/am/gateway/wso2am-pattern-2-am-gateway-ingress.yaml

```diff
@@ -36,3 +36,10 @@ spec:
             name: {{ template "am-pattern-2.resource.prefix" . }}-am-gateway-service
             port: 
               number: 8243
+      - path: /api/am/gateway/v1/
```

Comment:
I think it might be better to use a separate host for Gateway REST API since the `.Values.wso2.deployment.am.gateway.ingress.hostname` is used for APIs exposed through APIM

Reply:
Wouldn't that create additional confusion? two host names for gateway? With this approach, the downside is if someone uses '/api/am/gateway/v1/' as the context but I think the chances of creating that are slim.

adding @tharindu1st to get additional  comments

Reply:
This entry is removed